packages/katello/katello-repos/katello-repos.spec

```diff
@@ -1,11 +1,12 @@
 %global pulpcore_version 3.39
+%global candlepin_version 4.3
```

Comment:
I am not sure if I should start with 4.3 here or lean into the nightly version of the repository for `rpm/develop`.

Reply:
I'd be fine with 4.3 so we can pick it and then update to nightly in a separate bump. But no strong feelings here